vendor/github.com/docker/libnetwork/drivers/overlay/overlayutils/utils.go

```diff
@@ -0,0 +1,44 @@
+// Package overlayutils provides utility functions for overlay networks
+package overlayutils
+
+import (
+	"fmt"
+	"sync"
+)
+
+var (
+	vxlanUDPPort uint32
+	mutex        sync.Mutex
+)
+
+func init() {
+	vxlanUDPPort = 4789
+}
+
+// ConfigVxlanUDPPort configures vxlan udp port number.
+func ConfigVxlanUDPPort(vxlanPort uint32) error {
```

Comment:
Note that capitalization doesn't look correct (`VXLAN` is all caps, so this should probably be `ConfigVXLANUDPPORT`)

Reply:
I felt "ConfigVxlanUDPPort" is more readable than ConfigVXLANUDPPORT. If you are still not ok with it, I can change it.  pls let me know.

Reply:
> I felt "ConfigVxlanUDPPort" is more readable than ConfigVXLANUDPPORT. If you are still not ok with it, I can change it. pls let me know.

Yes, I definitely agree with you that `ConfigVXLANUDPPort` is ugly (due to both `VXLAN` and `UDP` being an accronym, so both are all-caps).

The reason I still would like to see the name changed (also in other locations), is that sticking to a convention makes the code more consistent, and removes "personal preference" for these (naming is already hard! :sweat_smile:).

The danger of "personal preference" (it "looks" better), is that it may trickle up into public API's; in this case; exported functions, but also the actual API.

For example, in our API we have;

- `CpuShares` in `/containers/<id>/json`, but `CPUShares` in `/info`,
- `NetworkSettings.IPAddress`, but [`NetworkSettings.Ports.HostIp`](https://github.com/moby/moby/blob/master/vendor/github.com/docker/go-connections/nat/nat.go#L16-L22)

Changing a public API / exported function once it's released, can be difficult, so ideally, they should be updated soon.

Reply:
Ok. Let me change it. I am seeing this comment only now.  Didn't see it 30 mins before.  Not sure if this github issue.